sycl/test/esimd/odr.cpp

```diff
@@ -12,7 +12,7 @@
 // UNSUPPORTED: cuda
 //
 // Linking issues with AMD:
-// XFAIL: rocm_amd
+// XFAIL: hip_amd
```

Comment:
This should rather be UNSUPPORTED: cuda,hip_amd, I believe, as esimd is only for Intel GPU.

Reply:
Oh I see, updated to `UNSUPPORTED: cuda || hip`

Reply:
I just realized `hip` is not a lit feature in this repo, it is in `llvm-test-suite` though, changed to `hip_amd` for now, the lit setup for this may require some refactoring in the future, it was a bit thrown off by the `on-device` tests move.